skin-schema.json

```diff
@@ -1174,6 +1204,26 @@
           "type": "string",
           "default": "icon icon-auto"
         }
+      },
+      "multiAudio": {
+        "description": "Set the styling for the multiaudio",
```

Comment:
Can we change this for "Set the styling for the multiaudio icon"?

Reply:
\+